src/devcenter/azext_devcenter/manual/_help.py

```diff
@@ -227,8 +227,8 @@
       - name: DevBoxes_DelayUpcomingAction
         text: |-
                az devcenter dev dev-box delay-upcoming-action --dev-center-name "{devCenterName}" \
---project-name "{projectName}" --delay-until "2022-09-30T17:00:00Z" --name "myDevBox" \
---upcoming-action-id "cae4d1f4-94b8-75f2-406d-5f00ae4c1da7" --user-id "me"
```

Comment:
nit: I know that we switched the default for user id to "me" but I think it's better to keep this parameter in the help example or at least specify a fake user id instead of me 

For example --user-id "00000000-0000-0000-0000-000000000000"